src/test/java/com/openlattice/rehearsal/SetupTestData.kt

```diff
@@ -36,42 +36,54 @@ open class SetupTestData : MultipleAuthenticatedUsersBase() {
     companion object {
         private const val DATA_FOLDER = "data"
         private const val FLIGHT_FOLDER = "flights"
+        private const val FLIGHT_SQL = "select * from public.socrates limit 10;"
+        private const val CONFIGURATION_KEY = "example_data"
 
         init {
             MissionControl.continueAfterSuccess()
         }
 
-        @Test
-        fun doImport() {
-            importDataSet("$DATA_FOLDER/socratesA.yaml","$FLIGHT_FOLDER/testdata1.csv")
-            importDataSet("$DATA_FOLDER/socratesB.yaml","$FLIGHT_FOLDER/testdata2.csv")
-        }
         /**
-         * Import datasets via Shuttle
+         * Import datasets from csv via Shuttle
          * @param
          */
-        fun importDataSet(flightFileName: String, dataFileName: String) {
+        fun importDataSet(flightFileName: String, dataSource: String, loadFromFile: Boolean) {
```

Comment:
Would prefer to make this two different methods that pass the necessary things into a private third.

```
private fun importDataSet(name, datasource, shuttleOptions: Array<String>)
fun importDataCSV(name, datasource) {
    ...
    importDataSet(name, datasource, arrayOf("-{CliOptions.CSV}=datafile")) 
}
fun importDataSQL(name, datasource) {
    ...
    importDataSet(name, datasource, arrayOf("sql...", "... options", etc))
}
```

boolean parameters make code way more mysterious to debug later.

Reply:
I think that's a bit weird, since all of those arguments are shuttle options, and the only thing importDataSet should do is then:
```
private fun importDataSet(shuttleOptions: Array<String>){
    main(shuttleOptions)
}
```
Which is longer than just adding `main()` within each function.